src/ssl.c

```diff
@@ -14072,6 +14072,22 @@ static WC_INLINE void RestoreSession(WOLFSSL* ssl, WOLFSSL_SESSION* session,
 #endif
 }
 
+static int SslSessionCacheOn(const WOLFSSL* ssl, const WOLFSSL_SESSION* session)
+{
+    (void)session;
+    if (ssl->options.sessionCacheOff
+    #if defined(HAVE_SESSION_TICKET) && defined(WOLFSSL_FORCE_CACHE_ON_TICKET)
+                && session->ticketLen == 0
+    #endif
+    #ifdef OPENSSL_EXTRA
+                && ssl->options.side != WOLFSSL_CLIENT_END
```

Comment:
Can you clarify why this check for server side only has been added now and wasn't there previously?

Reply:
I'm not exactly sure. I'll run the EAP-TLS tests with and without this to check if this is still necessary.